### ISSUE TITLE
RetroArch closes when trying to open non-existent core from the history tab.

### ISSUE THREAD

**Issue by orbea** [libretro/RetroArch#3761]
OS: `Slackware64-current`
`RetroArch-2016.10.08_f0ce438_master-x86_64-1_git`

If opening a core with that no longer exists, but still is shown in the xmb history tab RetroArch will immediately close, but no segfault. This can be annoying as its not always obvious which entry was a test core and which one is actually installed.

```
RetroArch [INFO] :: Loading dynamic libretro core from: "/media/gittings/installed/libretro/tmp/GLupeN64/glupen64_libretro.so"
RetroArch [ERROR] :: Failed to open libretro core: "/media/gittings/installed/libretro/tmp/GLupeN64/glupen64_libretro.so"
RetroArch [ERROR] :: Error(s): /media/gittings/installed/libretro/tmp/GLupeN64/glupen64_libretro.so: cannot open shared object file: No such file or directory
RetroArch [ERROR] :: Fatal error received in: "load_dynamic()"
RetroArch [ERROR] :: Failed to start audio driver. Will continue without audio.
RetroArch [ERROR] :: Failed to start audio driver. Will continue without audio.
```

**Comment by bparker06** [comment]
fixed in e49c31f.

**Comment by orbea** [comment]
Great, thanks!
